observing/obsstate.py

```diff
@@ -95,19 +97,30 @@ def read_calibrations():
 
 def add_session(sdffile: str):
     """Parse SDF to create and add new session to the database."""
+
     assert os.path.exists(sdffile), f"{sdffile} does not exist"
     dd = parsesdf.sdf_to_dict(sdffile)
+    now = Time.now()
     # convert lists to comma-separated strings
     for key, value in dd['SESSION'].items():
         if isinstance(value, list):
             dd['SESSION'][key] = ', '.join(map(str, value))
-    session = Session(**dd['SESSION'], STATUS='scheduled')
+
+    session = Session(**dd['SESSION'], time_loaded=now.mjd, STATUS='scheduled')
```

Comment:
If you're storing timestamp as string (as you are), feed it the isot string? I don't like mjd since it doesn't do leap seconds....but if you're feeding it mjd, use real, and if unix timestamp, integer.